web/app/help/operations/import-from-neo4j.asciidoc

```diff
@@ -0,0 +1,29 @@
+### Import from Neo4j
+
+Import a graph from the https://neo4j.com/[Neo4j] graph database.
+
+Neo4j does not have a strict schema. Different nodes may have different attributes.
+In LynxKite the list of vertex attributes is defined for the whole graph.
+But each vertex may leave any attribute undefined.
+
+If you import Neo4j nodes that have different attributes, such as movies that have
+a `title` and actors that have a `name`, the resulting graph will have both `title` and
+`name` attributes. `title` will only be defined on movies, `name` will only be defined
+on actors.
```

Comment:
I think we may need to explain that there can be problems if the same attribute has different type on different kinds. And that the way to fix that is to make the query more specific somehow and not import (both) of the conflicting attributes.